src/Testing/PHPUnit/ValueObject/RectorTestResult.php

```diff
@@ -30,7 +30,7 @@ public function getAppliedRectorClasses(): array
     {
         $rectorClasses = [];
 
-        foreach ($this->processResult->getFileDiffs() as $fileDiff) {
+        foreach ($this->processResult->getFileDiffs(false) as $fileDiff) {
```

Comment:
Here we want to get all file diffs, including those where there was no actual change